.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_KEY = AIzaSyCvXyI17JpqteOyWUvL_w7gjwXKOZXnbaI
```

Comment:
Normally this type of secret needs to be added to the `.gitignore` file for security reasons.